tests/update_testapps_dependencies.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Update dapr dependencies for all e2e test apps
+cd ./apps
+for i in `ls`
+do
+   if test -f "$i/go.mod"
+   then
```

Comment:
We do not need to use `ls`. bash for can iterate directories or files itself.
e.g.
```
for file in */; do
    if [ -f "$file/go.mod" ]; then
        # update module
    fi
done
```

Reply:
fixed